www/docs/sidebars.js

```diff
@@ -214,6 +214,17 @@ module.exports = {
                 },
               },
             },
+{
+                            type: 'doc',
+                            id: 'deployments/server/deploying-on-microtica',
+                            label: 'Deploy on Microtica',
+                            customProps: {
+                                themedImage: {
+                                    light: 'https://res.cloudinary.com/dza7lstvk/image/upload/v1681296578/Medusa%20Docs/Other/aF4ZuDS_t11rcu.png',
+                                    dark: 'https://res.cloudinary.com/dza7lstvk/image/upload/v1681296612/Medusa%20Docs/Other/Lg7NHQk_qo6oax.png'
+                                }
+                            }
+                        }
```

Comment:
Can you place this before the Qovery guide as it's not labeled as deprecated?

Reply:
done